india_forecast_app/models/pvnet/model.py

```diff
@@ -189,6 +192,15 @@ def predict(self, site_id: str, timestamp: dt.datetime):
                 values_df["forecast_power_kw"].rolling(4, min_periods=1).mean().astype(int)
             )
 
+        if self.smooth_blocks:
```

Comment:
I just noticed there is some smoothing that happens just above this using 1 hour windows, does this mean this is this smoothing is applied on top of that one?

Feels like it would be useful have a unit test with non random forecast values to make sure this is working as expected